gdeploy.spec

```diff
@@ -37,16 +39,15 @@ See http://gdeploy.readthedocs.io/en/latest/ for more details
 # ...
 
 %build
-%{__python2} setup.py build
-%if 0%{?with_doc}
+%py2_build
```

Comment:
%if 0%{?with_doc} has been removed, causing the following %endif to be without an %if